avaframe/com1DFA/com1DFA.py

```diff
@@ -397,16 +471,23 @@ def prepareInputData(inputSimFiles):
     releaseLine['type'] = 'Release'
 
     # get line from secondary release area polygon
-    if entResInfo['flagSecondaryRelease'] == 'Yes':
-        secondaryReleaseFile = inputSimFiles['secondaryReleaseFile']
-        secondaryReleaseLine = shpConv.readLine(secondaryReleaseFile, '', demOri)
-        secondaryReleaseLine['fileName'] = [secondaryReleaseFile]
-        secondaryReleaseLine['type'] = 'Secondary release'
+    if cfg.getboolean('secRelArea'):
+        if entResInfo['flagSecondaryRelease'] == 'Yes':
+            secondaryReleaseFile = inputSimFiles['secondaryReleaseFile']
+            secondaryReleaseLine = shpConv.readLine(secondaryReleaseFile, '', demOri)
+            secondaryReleaseLine['fileName'] = [secondaryReleaseFile]
+            secondaryReleaseLine['type'] = 'Secondary release'
+        else:
+            message = 'No secondary release file found'
+            log.error(message)
+            raise FileNotFoundError(message)
     else:
         secondaryReleaseLine = None
+        # set False
+        entResInfo['flagSecondaryRelease'] = 'No'
```

Comment:
I suggest using `False` insted of `'No'`

Reply:
these 'flags' are set to No and Yes for also entrainment and resistance as they are then directly stated in the markdown report - and initially chosen to be more readable as yes and no instead of true and false - should we keep this or change it throughout? but this has an effect on many places so if we want to change it I suggest making an issue

Reply:
Didn't realize the report connection, keep it